src/components/Icon/Icon.scss

```diff
@@ -0,0 +1,31 @@
+.Icon {
+    display: inline-flex;
+    align-self: center;
+
+    &--margin {
+        @each $side in (top, right, bottom, left) {
+            &_#{ $side } {
+                margin-#{ $side }: .25em;
+            }
+        }
+    }
+
+    &__svg {
+        position: relative;
+        top: .125em;
+        width: 1em;
+        height: 1em;
+    }
+
+    &__path {
+        transition: fill var(--transition-duration);
+
+        &--context {
+            @each $context in (neutral, brand, primary, accent, info, good, warning, bad) {
+                &_#{ $context } {
+                    fill: var(--color-#{ $context });
```

Comment:
Color can be handled by top level component and here we can use `fill: currentColor` (or even nothing is needed to inherit fill value from a parent)

Reply:
If you can apply `fill` on the SVG instead of each path while preserving the current aesthetics then it makes sense indeed